src/main/java/cz/muni/fi/Component.java

```diff
@@ -0,0 +1,62 @@
+
+package cz.muni.fi;
+
+import jdk.nashorn.internal.objects.annotations.Getter;
+import jdk.nashorn.internal.objects.annotations.Setter;
+
+import javax.persistence.Enumerated;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+/**
+ * Author: Richard Hrmo
+ */
+public class Component {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Getter @Setter private Long id;
+
+    @Getter private String name;
+
+    @Getter private boolean isAvailable = false;
```

Comment:
Could we simplify this to available instead of isAvailable?

Reply:
isAvailable je nazov ktory mi dala IDEA, takto sa maju oznacovat boolean co viem ja, kvoli tomu aby si hned vedel co dana premenna znamena